olga/charts/views.py

```diff
@@ -2,78 +2,59 @@
 Views for the charts application.
 """
 
-from __future__ import division
-
+import datetime
 import json
 
-import pycountry
 from django.shortcuts import render
 from django.views.generic import View
 
 from olga.analytics.models import InstallationStatistics
 
 
-class MapView(View):
+def get_first_and_last_datetime_of_update_data():  # pylint: disable=invalid-name
     """
-    Displays information on a world map and tabular view.
+    Get first and last datetimes OLGA acceptor gathers statistics.
     """
-
-    @staticmethod
-    def get(request):
-        """
-        Passes graph data to frontend.
-        """
-
+    try:
         first_datetime_of_update_data = InstallationStatistics.objects.first().data_created_datetime
         last_datetime_of_update_data = InstallationStatistics.objects.last().data_created_datetime
+    except AttributeError:
```

Comment:
i think it is bit wrong. If you will always get first and last, then it is useless. But if somebody from them is None, that means that one or 0 objects in database, then logic flow is unclear. Lets make it explicit.